sig-architecture/production-readiness.md

```diff
@@ -28,6 +27,28 @@ aspects of the process.
     happens if it is subsequently upgraded again?
   - Are there tests for this?
 * Scalability
+  - Does enabling/using the feature will result in any new API calls?
+    Describe them with their impact keeping in mind the [supported limits][]
+    (e.g. 5000 nodes per cluster, 100 pods/s churn) focusing mostly on:
+     - components listing and/or watching resources they didn't before
+     - API calls that may be triggered by changes of some Kubernetes
+       resources (e.g. update object X based on changes of object Y)
+     - periodic API calls to reconcile state (e.g. periodic fetching state,
+       heartbeats, leader election, etc.)
+  - Does enabling/using the feature will result in supporting new API
+    objects? How many of those will be supported (and how that translates
+    to limitations for users)?
+  - Does enabling/using the feature will result in increasing size or count
+    of the existing API objects?
+  - Does enabling/using the feature will result in increasing time taken
+    by any operations covered by [existing SLIs/SLOs][] (e.g. by adding
+    additional work, introducing new steps in between, etc.)?
+    Please describe the details if so.
+  - Does enabling/using the feature will result in non-negligible increase
+    of resource usage (CPU, RAM, disk IO, ...) in any components?
+    Things to keep in mind include: additional in-memory state, additional
+    non-trivial computations, excesive access to disks, significant amount
+    of data sent and/or received over network, etc.
```

Comment:
I really like these examples, increasing log volume might be also a good one when it comes to disk IO.

Reply:
done